pkg/web/src/app/Providers/components/ProvidersTable/ProviderActionsDropdown.tsx

```diff
@@ -11,6 +16,21 @@ import { isSameResource } from '@app/queries/helpers';
 import { useHistory } from 'react-router-dom';
 import { useClusterProvidersQuery } from '@app/queries';
 
+export const hasRunningMigration = ({
+  plans = [],
+  providerMetadata,
+}: {
+  plans: IPlan[];
+  providerMetadata: INameNamespaceRef;
+}) =>
```

Comment:
Add return type of boolean just to be more explicit in case one misses the `!!` bit of `!!plans`?